code/cubical/Examples/Indexed/Dyck.agda

```diff
@@ -127,6 +152,52 @@ data TraceTag (b : Bool) (n : ℕ) : Type where
   leftovers' : (b Eq.≡ false) → ∀ n-1 → (suc n-1 Eq.≡ n) → TraceTag b n
   unexpected' : b Eq.≡ false → n Eq.≡ zero → TraceTag b n
 
+isSetTraceTag : ∀ b n → isSet (TraceTag b n)
+isSetTraceTag b n = isSetRetract {B = TT} enc dec retr isSetTT where
+  isPropBoolEq : ∀ {b b' : Bool} → isProp (b Eq.≡ b')
+  isPropBoolEq {b}{b'} =
+    subst isProp Eq.PathPathEq (isSetBool _ _)
+
+  isPropℕEq : ∀ {b b' : ℕ} → isProp (b Eq.≡ b')
+  isPropℕEq {b}{b'} =
+    subst isProp Eq.PathPathEq (isSetℕ _ _)
+
+  TT : Type ℓ-zero
```

Comment:
After reviewing the rest, I'm going to rename this bc it looks like `tt : Unit`

Reply:
it's a a private local definition so I didn't really care much about the name

Reply:
fair enough, I'll just leave it then